lstchain/scripts/lstchain_longterm_dl1_check.py

```diff
@@ -792,8 +794,17 @@ class pixwise_info(tables.IsDescription):
 
     # We write out the camera geometry information, assuming it is the same
     # for all files (hence we take it from the first one):
-    subarray_info = SubarrayDescription.from_hdf(files[0])
-    subarray_info.to_hdf(output_file_name)
+    # subarray_info = SubarrayDescription.from_hdf(files[0])
+    # subarray_info.to_hdf(output_file_name)
+
+    cam_description_table = \
```

Comment:
Style: please don't use explicit line continuations, just use the implicit splits using parentheses:

```
cam_description_table = Table.read(
    files[0],
    path='/configuration/instrument/telescope/camera/geometry_0'
)
```

But in this case, please just use `subarray = SubarrayDescription.load(files[0])`, please don't rely on `geometry_0` being the LST.